src/structures/guild_preference.ts

```diff
@@ -230,7 +233,49 @@ export default class GuildPreference {
     /** @returns a friendly, potentially truncuated, string displaying the currently selected exclude option */
     getDisplayedExcludesGroupNames(): string {
         if (this.gameOptions.excludes === null) return null;
-        let displayedGroupNames = this.gameOptions.excludes.map((x) => x.name).join(", ");
+        let displayedGroupNames = this.gameOptions.excludes.map((x) => x.name)
+            .filter((name) => !name.includes("+"))
+            .join(", ");
+        if (displayedGroupNames.length > 400) {
+            displayedGroupNames = `${displayedGroupNames.substr(0, 400)} and many others...`;
```

Comment:
Potentially convert this (including the filter, the map, etc.) into a function you can reuse in groups, includes